src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -552,17 +552,7 @@ private MulticlassLogisticRegressionPredictor(IHostEnvironment env, ModelLoadCon
             if (ctx.TryLoadBinaryStream(LabelNamesSubModelFilename, r => labelNames = LoadLabelNames(ctx, r)))
                 _labelNames = labelNames;
 
-            string statsDir = Path.Combine(ctx.Directory ?? "", ModelStatsSubModelFilename);
-            using (var statsEntry = ctx.Repository.OpenEntryOrNull(statsDir, ModelLoadContext.ModelStreamName))
-            {
-                if (statsEntry == null)
-                    _stats = null;
-                else
-                {
-                    using (var statsCtx = new ModelLoadContext(ctx.Repository, statsEntry, statsDir))
-                        _stats = LinearModelStatistics.Create(Host, statsCtx);
-                }
-            }
+            ctx.LoadModelOrNull< LinearModelStatistics, SignatureLoadModel>(Host, out _stats, ModelStatsSubModelFilename);
```

Comment:
>LoadModelOrNull [](start = 16, length = 15)

Hi @sfilipi, you did not do this I know, but I know you're also touching this code here as part of some other work you're doing. Perhaps you also noticed this strange code and fixed it too, not sure.

Reply:
It didn't catch my eye, but thanks for the note.